config/default/jenkins-infra.yaml

```diff
@@ -0,0 +1,127 @@
+                                resourceLimitMemory: "512Mi"
+                                resourceRequestCpu: "500m"
+                                resourceRequestMemory: "512Mi"
+                                args: "^${computer.jnlpmac} ^${computer.name}"
+                                alwaysPullImage: true
+                          - name: jnlp-windows
+                            nodeSelector: "kubernetes.io/os=windows"
+                            containers:
+                              - name: jnlp
+                                image: "jenkins4eval/jnlp-agent:latest-windows"
+                                resourceLimitCpu: "500m"
+                                resourceLimitMemory: "512Mi"
+                                resourceRequestCpu: "500m"
+                                resourceRequestMemory: "512Mi"
+                                args: "^${computer.jnlpmac} ^${computer.name}"
+                                alwaysPullImage: true
+                            label: "jnlp-windows"
+                            yaml: |-
+                              spec:
+                                tolerations:
+                                - key: "os"
+                                  operator: "Equal"
+                                  value: "windows"
+                                  effect: "NoSchedule"
+              jobs-settings: |
+                jobs:
+                  - script: >
+                      multibranchPipelineJob('k8smgmt') {
+                        displayName "K8s Cluster Management"
+                        description "Current K8s Cluster Management"
+                        branchSources {
+                          github {
+                            id('2019081602')
+                            scanCredentialsId('github')
+                            repoOwner('jenkins-infra')
+                            repository('charts')
+                          }
+                        }
+                        factory {
+                          workflowBranchProjectFactory {
+                            scriptPath('Jenkinsfile_k8s')
+                          }
+                        }
+                      }
+              ldap-settings: |
+                jenkins:
+                  securityRealm:
+                    ldap:
+                      configurations:
+                        - server: "${LDAP_SERVER}"
+                          rootDN: "${LDAP_ROOT_DN}"
+                          managerDN: "${LDAP_MANAGER_DN}"
+                          managerPasswordSecret: "${LDAP_MANAGER_PASSWORD}"
+                          userSearch: cn={0}
+                      cache:
+                        size: 100
+                        ttl: 300
+              matrix-settings: |
+                jenkins:
+                  authorizationStrategy:
+                    globalMatrix:
+                      grantedPermissions:
+                        - "Overall/Administer:admins"
```

Comment:
what about?
```suggestion
                        - "Overall/SystemRead:all"
```

Reply:
Would it be safe to enabled the systemread by default?